browser/tor/tor_profile_service_impl.cc

```diff
@@ -123,10 +124,12 @@ TorProfileServiceImpl::TorProfileServiceImpl(Profile* profile)
   tor_launcher_factory_ = TorLauncherFactory::GetInstance();
   tor_launcher_factory_->AddObserver(this);
 
+#if BUILDFLAG(ENABLE_TOR)
```

Comment:
TorProfileServiceImpl should definitely not be built at all if tor is not enabled in gn

Reply:
fixed in 9bb8d68ed53a77079155beab160dea6807ee93b8